CHANGELOG.md

```diff
@@ -6,11 +6,6 @@
 
```

Comment:
No need to touch this flie. It is autogenerated (it will update in the next point release)

Reply:
Ok, I'll revert the changes